components/camera/client.go

```diff
@@ -201,13 +146,19 @@ func (c *client) Stream(
 				return
 			}
 
-			frame, release, err := c.Read(streamCtx)
+			resBytes, resMetadata, err := c.Image(streamCtx, mimeTypeFromCtx, nil)
 			if err != nil {
 				for _, handler := range errHandlers {
 					handler(streamCtx, err)
 				}
 			}
 
+			img, err := rimage.DecodeImage(ctx, resBytes, resMetadata.MimeType)
```

Comment:
We are not using `GetGoImage` here because of specific error handler?

Reply:
Yes that's why I left it. Although on a second pass, we could still use it. Just pass the error the handler. I'll try that out

Reply:
Chagned. Let me know how this looks